packages/unity-bootstrap-theme/stories/molecules/heroes/heroes.examples.stories.js

```diff
@@ -23,99 +23,134 @@ export const HeroMedium = createStory(
       fetchpriority="high"
     />
     <h1>
-      <span className="highlight-white">
-        Lorem ipsum dolor sit amet, consectetur adip
+      <span className="highlight-gold">
+        ASU named to prestigious university association
```

Comment:
We need less words in this hero for the example. 3-4 words would be ideal - maybe "ASU named prestigious university".